packages/relay/src/lib/models/account.ts

```diff
@@ -0,0 +1,43 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { Balance } from "./balance"
+import { Key } from "./key";
+
+export interface Account {
```

Comment:
For Mirror Node models I don't think we should add the dependency.
This will force us to have to maintain the model and changes.
Initially I thought eventually we'd push it back to the Mirror Node team so they expose an npm model package, however, I was reminded we did all the work around OpenAPI spec so that projects could use it for consideration like this.

I think we should explore something like https://github.com/swagger-api/swagger-js/blob/master/docs/usage/http-client-for-oas-operations.md that takes the Mirror Node Open AI Spec and generates model for us.
Then all we have to do is upgrade version to get the new details.
We can explore this or other model generators but we should only depend on the Open API Spec

Reply:
Gotcha, I've initially thought about mirror node team exposing a npm model package, but figured that this won't happen anytime soon. Will research the proposed solutions, closing this PR.

Reply:
I've created a ticket issue for tracking this, also added some options that found. 
https://github.com/hashgraph/hedera-json-rpc-relay/issues/1416